tests/cases/crashes/layout_deleted_item.slint

```diff
@@ -64,9 +64,9 @@ assert!(!instance.get_hover());
 
 ```js
 var instance = new slint.TestCase({
-    clicked: function() { var x = instance.model; x.pop(); instance.model = x; }
+    clicked: function() { var x = instance.model; x.remove(x.length - 1, 1); instance.model = x; }
```

Comment:
Do the change in the tests means we broke compatibility?

Reply:
Yes that's true. See next comment for details.